From 34412e8e3d59a45e039ee9f2bdbc709c45b51814 Mon Sep 17 00:00:00 2001
From: "Gille, Marc" <Marc.Gille@sungard.com>
Date: Mon, 21 Jul 2014 11:26:10 -0400
Subject: [PATCH 6/6] Minor adjustments.

---
 .../js/BusinessObjectManagementPanelController.js  | 26 ----------------------
 1 file changed, 26 deletions(-)

diff --git a/stardust-business-object-management/src/main/resources/META-INF/webapp/js/BusinessObjectManagementPanelController.js b/stardust-business-object-management/src/main/resources/META-INF/webapp/js/BusinessObjectManagementPanelController.js
index 7de8002..b3a7638 100644
--- a/stardust-business-object-management/src/main/resources/META-INF/webapp/js/BusinessObjectManagementPanelController.js
+++ b/stardust-business-object-management/src/main/resources/META-INF/webapp/js/BusinessObjectManagementPanelController.js
@@ -46,7 +46,6 @@ define(
 							});
 
 					return deferred.promise();
-
 				};
 
 				/**
@@ -113,30 +112,5 @@ define(
 										self.rootController.safeApply();
 									}).fail();
 				};
-
-				/**
-				 * 
-				 */
-				BusinessObjectManagementPanelController.prototype.onBusinessObjectInstanceSelectionChange = function() {
-					// var self = this;
-					//
-					// jQuery("*").css("cursor", "wait");
-					//
-					// DocumentAssignmentService.instance().getPendingProcesses()
-					// .done(function(pendingProcesses) {
-					// self.pendingProcesses = pendingProcesses;
-					//
-					// self.refreshPendingProcessesTree();
-					// self.safeApply();
-					//
-					// jQuery("*").css("cursor", "default");
-					//
-					// window.setTimeout(function() {
-					// self.bindDragAndDrop();
-					// }, 1000);
-					// }).fail(function() {
-					// jQuery("*").css("cursor", "default");
-					// });
-				};
 			}
 		});
-- 
1.9.4.msysgit.0

